src/fal/cli/flow_runner.py

```diff
@@ -83,47 +84,41 @@ def node_to_script(node: Union[FalFlowNode, None], fal_dbt: FalDbt) -> FalScript
         raise Exception(f"Cannot convert node to script. Node: {node}")
 
 
-def _combine_fal_run_results(target_path):
-    result_files = _get_fal_run_results(target_path)
+def _combine_fal_run_results(target_path: str) -> None:
+    target_path = Path(target_path)
+    dbt_run_results, fal_run_results = [], []
+    for path in target_path.glob("fal_results_*.json"):
+        assert path.is_file()
```

Comment:
What would it mean for a path to not be a file? Should we catch this more gently?

Reply:
> What would it mean for a path to not be a file? Should we catch this more gently?

I don't think so, this is just a sanity check (you can create a directory named `fal_results_1.json` but I don't think it should be a case we should try to handle).